pegasus/test/test_deliverer_render.rb

```diff
@@ -0,0 +1,100 @@
+require_relative 'test_helper'
+require 'cdo/poste'
+
+class FakeSmtp
```

Comment:
I don't think this class is necessary for the render test; certainly the `attr_reader` logic is not used

Reply:
yep, you're right, updated